slack/views.py

```diff
@@ -105,3 +109,31 @@ def cron_minute(request):
     "Handles actions that need to take place every minute"
     handle_notifications()
     return HttpResponse()
+
+def claim_slack_user(request, user_id): # Me UHMBFACG5
+    slack_user = SlackUser.objects.get(user_id=user_id)
+    if slack_user.owner:
+        return HttpResponseForbidden("User was already claimed")
+
+    token = dumps({
+        'created': datetime.now().isoformat(),
+        'slack_user_id': slack_user.user_id })
+
+    url = request.build_absolute_uri(reverse("confirm_claim_slack_user")) + f'?token={token}'
+    send_message(slack_user.user_id,f'Would you like to link your slack account with response user `{request.user.username}` <{url}|Link account>')
+
+    return HttpResponse('')
+
+def confirm_claim_slack_user(request):
```

Comment:
Would it be worth adding something to the README to explain that users will need to do this one time operation for setup?

Reply:
Can you share some screenshots of the flow a user has to go through to claim their Slack ID?